### PR TITLE
Feature/update post request

### PR DESCRIPTION
**Pull request by dlouise64**
# What has changed and why?
UI Changes
Autocomplete form
Populate other fields (name)
Populate applicant total rates
Send data to second form page and auto populate there.

Some styles changes still required, will update after this PR

# How to test this change

- [ ] has automated tests
- [ ] at least one a reviewer ran the code
- [ ] tested in small screen
- [ ] tested in Chrome
- [ ] tested in Firefox
- [ ] tested in IE
- [ ] tested in Safari
- [ ] i18n
- [ ] tested in screen reader/contrast <-- remove this if no UX change

### CODE REVIEW

File: src/styles/scss/App.scss [hunk 1]
```diff
@@ -350,3 +350,24 @@ p {
 canvas {
   border: 1px solid #000;
 }
+
+.Select-input > input {
+  width: 80% !important;
```
[the remark this answers is not in the document]
Comment (dlouise64) [reply src/styles/scss/App.scss]: @Br3nda this is overwriting a packages styles so its needed

**Comment by Br3nda** [comment]
The 2x green lines makes me think i need to keep typing. Can it be only one that you can edit?

image

**Comment by Br3nda** [comment]
it says i'm eligible before i've entered enough data for it to know that. Can it be hidden?
image

**Comment by Br3nda** [comment]
The dependents entry doesn't look like an input box -- there's no green underline
image

**Comment by Br3nda** [comment]
Nulls coming through in names, this is when the firstnames are null.

image

**Comment by Br3nda** [comment]
Lots of things working! Lots of populating the right fields
i was able to apply and sign.

**Pull request merged**